inlong-sort/sort-connectors/jdbc/src/main/java/org/apache/inlong/sort/jdbc/internal/JdbcMultiBatchingOutputFormat.java

```diff
@@ -506,13 +508,18 @@ protected void attemptFlush() throws IOException {
                             }
                         }
                     }
+                    if (!flushFlag && null != tableException) {
+                        LOG.info("Put tableIdentifier:{} exception:{}",
+                                tableIdentifier, tableException.getMessage());
+                        tableExceptionMap.put(tableIdentifier, tableException);
+                        if (isIgnoreTableException) {
+                            LOG.info("Stop write table:{} because occur exception",
+                                    tableIdentifier);
+                            break;
```

Comment:
`break` is right?

Reply:
break and continue both ok.  continue is better and modifed to continue.